script/hipify-inplace.sh

```diff
@@ -35,32 +38,36 @@ declare -a all_srcs=(
     ${hipify_perl_srcs[@]} $(find_code tensoradapter graphbolt)
 )
 
+echo "Starting hipify jobs"
 declare -a log_files=()
 
-log_file="$(mktemp --tmpdir hipify_tensoradapter.XXX.log)"
+log_file="$(mktemp --tmpdir="${LOG_DIR}" hipify_tensoradapter.XXX.log)"
 log_files+=("${log_file}")
-( set -x ; script/hipify-torch-extension.py ${DGL_HOME}/tensoradapter/ &> "${log_file}" ) &
+( set -x ; script/hipify-torch-extension.py ${DGL_HOME}/tensoradapter/ ) &> "${log_file}" &
 
-log_file="$(mktemp --tmpdir hipify_graphbolt.XXX.log)"
+log_file="$(mktemp --tmpdir="${LOG_DIR}" hipify_graphbolt.XXX.log)"
 log_files+=("${log_file}")
-( set -x ; script/hipify-torch-extension.py ${DGL_HOME}/graphbolt/ &> "${log_file}" ) &
+( set -x ; script/hipify-torch-extension.py ${DGL_HOME}/graphbolt/ ) &> "${log_file}" &
 
 
 for src in ${hipify_perl_srcs[@]}; do
-    log_file="$(mktemp --tmpdir hipify_${src//\//_}.XXX.log)"
+    log_file="$(mktemp --tmpdir="${LOG_DIR}" hipify_${src//\//_}.XXX.log)"
     log_files+=("${log_file}")
-    ( set -x ; hipify-perl -print-stats -inplace $src &> "${log_file}" ) &
+    ( set -x ; hipify-perl -print-stats -inplace $src ) &> "${log_file}" &
 done
 
 sleep 1 # Hack to make it more likely the echo below prints after the last job command line from above.
```

Comment:
Not related to this commit, but should we make it `wait`?

Reply:
It does wait right below. The issue is that the message saying it was waiting was sometimes showing up before the echo'ed command line from the last job. But actually since we aren't echoing the job command lines, this can be removed entirely.